pkg/virt-controller/watch/vmi.go

```diff
@@ -2341,3 +2357,43 @@ func generateInterfaceStatusPatchRequest(oldInterfaceStatus []byte, newInterface
 		fmt.Sprintf(`{ "op": "add", "path": "/status/interfaces", "value": %s }`, string(newInterfaceStatus)),
 	}
 }
+
+func (c *VMIController) isCPUHotplugReady(vmi *virtv1.VirtualMachineInstance, pod *k8sv1.Pod) (bool, error) {
```

Comment:
Maybe I'm missing something, but I don't really understand the purpose of this commit in general.
We already have a condition, why do we need another label?

I mean, if you're extracting virt-controller's function, why not to extract them to a package that's also imported by virt-handler, then remove the condition only from virt-handler itself.

Reply:
@enp0s3 Please leave a commit message and comment here explaining why it is necessary. Also, let's cover this with tests.